### ISSUE TITLE
Allow offset to have separate configuration for AnimateIn and AnimateOut

### ISSUE THREAD

**Issue by xscottxbrownx** [MetinArslanturk/react-animation-on-scroll#11]
It would be helpful if we could have separate `offset` amounts for `animateIn` and `animateOut` - mostly when using sticky navbar.

**Comment by MetinArslanturk** [comment]
Makes sense. At the moment I'm a bit busy with some personal projects. But I am definetely open for contributions. Otherwise, I will maintain this lib probably next month.
